ci/jenkins/test.sh

```diff
@@ -595,10 +605,12 @@ function deliver_antrea {
             ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n jenkins@${IP} "${CLEAN_STALE_IMAGES_CONTAINERD}; ${PRINT_CONTAINERD_STATUS}; ctr -n=k8s.io images import ${DEFAULT_WORKDIR}/antrea-ubuntu.tar; ctr -n=k8s.io images import ${DEFAULT_WORKDIR}/flow-aggregator.tar" || true
         done
     elif [[ $TESTBED_TYPE == "kind" ]]; then
-            kind load docker-image antrea/antrea-agent-ubuntu:latest --name ${KIND_CLUSTER}
-            kind load docker-image antrea/antrea-controller-ubuntu:latest --name ${KIND_CLUSTER}
+            kind load docker-image antrea/antrea-agent-ubuntu:$CUSTOM_BUILD_TAG --name ${KIND_CLUSTER}
+            kind load docker-image antrea/antrea-controller-ubuntu:$CUSTOM_BUILD_TAG --name ${KIND_CLUSTER}
             kind load docker-image antrea/flow-aggregator:latest --name ${KIND_CLUSTER} 
             kubectl config use-context kind-${KIND_CLUSTER}
+            sed -i "s|antrea-agent-ubuntu:latest|antrea-agent-ubuntu:${CUSTOM_BUILD_TAG}|g" ./build/yamls/antrea.yml
+            sed -i "s|antrea-controller-ubuntu:latest|antrea-controller-ubuntu:${CUSTOM_BUILD_TAG}|g" ./build/yamls/antrea.yml
```

Comment:
Let's invoke `./hack/generate-manifest.sh` with the right parameters instead. I'd rather limit the use of `sed` for this.